packages/i18n-utils/src/localize-url.tsx

```diff
@@ -165,42 +211,20 @@ export function localizeUrl(
 		return fullUrl;
 	}
 
-	// Ignore and passthrough /relative/urls that have no host specified
 	if ( url.origin === INVALID_URL ) {
-		return fullUrl;
+		return localizePath( fullUrl, locale, isLoggedIn, url );
 	}
 
 	// Let's unify the URL.
 	url.protocol = 'https:';
 	// Let's use `host` for everything.
 	url.hostname = '';
```

Comment:
I haven't been able to get this to work. It seems like assigning an invalid `hostname` is ignored.

Reply:
Originally came from over here https://github.com/Automattic/wp-calypso/pull/38683/files?w=1 I must have changed it to `hostname = ''` in the move. Could just delete the line?